src/BaseTable.js

```diff
@@ -163,7 +165,7 @@ class BaseTable extends React.Component {
     const columns = this.getColumns();
 
     return (
-      <Table className={tableClassName} style={tableStyle} key="table">
+      <Table className={tableClassName} style={tableStyle} key="table" {...dataOrAriaProps}>
```

Comment:
Why not pass them to https://github.com/react-component/table/blob/479bd2ec10e8a7d8d0818b072f769ab7d5c7b1af/src/Table.js#L497-L502

Reply:
My thought is that attributes would be assigned on the native elements that these components represent, when available. On `Input` it would be on `<input>`, on `Table` it would go to the `<table>`. Others like `Select` don't use the native html elements, so then it would go to whatever wrapper makes sense.

Reply:
But I see your point, thats where `props.id` is assigned, so it might make the most sense.